modules/virt-about-node-placement-virtualization-components.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * virt/install/virt-specifying-nodes-for-virtualization-components.adoc
+
+[id="virt-about-node-placement-virtualization-components_{context}"]
+= About node placement for virtualization components
+
+You might want to customize where {VirtProductName} deploys its components to ensure that:
+
+* Virtual machines only deploy on nodes that are intended for virtualization workloads.
+* Operators only deploy on infrastructure nodes.
+* Certain nodes are unaffected by {VirtProductName}. For example, you have workloads unrelated to virtualization running on your cluster, and you want those workloads to be isolated from {VirtProductName}.
+
+You can specify node placement rules for the {VirtProductName} Operators that Operator Lifecycle Manager (OLM) deploys by editing the OLM `Subscription` object.
+
+To specify node placement rules for components that the {VirtProductName} Operators deploy, edit the HyperConverged Cluster custom resource (CR).
+
+[id="about-node-placement-rules_{context}"]
+== About node placement rules
+
+`nodeSelector`:: Allows pods to be scheduled on nodes that are labeled with the key-value pair or pairs that you specify in this field. The node must have labels that exactly match all listed pairs.
```

Comment:
Since all these rules are not KubeVir or HCO terms, but known API of K8s, I would add a link to K8s/OpenShift documentation for each one of the three rules. 

I know there are "additional resources" links below, and it's great, but I think many users just search for a specific solution for their current issue they need to solve, so I think as a reader, I would expect to have these links within the text itself in addition to the references at the bottom of the page.

Reply:
Great point. I moved the additional resources links up to below the "about" module, which is the best I can do given our current rules (no xrefs in modules).